core/src/main/java/com/datastax/oss/driver/api/core/metadata/Node.java

```diff
@@ -15,6 +15,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/*
```

Comment:
I don't understand this change - it's inconsistent - in some files it was added, in others it wasn't.